extensions/aws-ec2/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -31,3 +31,4 @@ metadata:
   - "integration"
   status:
   - "stable"
+  - "deprecated"
```

Comment:
These deprecation changes would deserve a separate commit. They belong to the Camel 3.8 upgrade.

Reply:
@ppalaga  I've put deprecation changed into separated commit in this PR, should I create another PR?

Reply:
This commit is OK